src/types/tableTypes.ts

```diff
@@ -0,0 +1,10 @@
+export type ColumnDefinition = {
+  title: string;
+  key: string;

```

Comment:
Tady bych přidal ještě něco jako cellFactory, která dostane value a vrátí JSX - občas chceš do tabulky nějaký custom JSX, jako je odkaz, pill atp. Pokud bude cellFactory undefined, tak tam vrátíš jen čistá data.

Reply:
Jo, něco takového mám určitě v plánu. Plus nějak nadefinovat, jestli se podle toho dá řadit, filtrovat (případně jak filtrovat)...

Reply:
Ideálně přidělej rovnou s tím prvním screenem, aby tam mohl být proklik na detail zřizovatele. Tady to klidně pošli dál a přidej v dalším PR.

Reply:
ok, přidal jsem